<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-20485NC4" ignored="0" name="Default test" tests="3" failures="1" timestamp="22 Apr 2020 07:31:05 GMT" time="16.619" errors="0">
  <testcase name="runScenario" time="8.032" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="4.193" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="3.582" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [300] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [300] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at stepDefinition.ApiTestingStepDef.i_verify_the_status_code_as(ApiTestingStepDef.java:91)
at âœ½.I verify the status code as "300"(file:src/test/resources/features/APITesting_GetOrder.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
